openfpga_flow/openfpga_shell_scripts/read_unique_blocks_full_flow_example_script.openfpga

```diff
@@ -0,0 +1,79 @@
+# Run VPR for the 'and' design
+#--write_rr_graph example_rr_graph.xml
+vpr ${VPR_ARCH_FILE} ${VPR_TESTBENCH_BLIF} --device ${OPENFPGA_VPR_DEVICE} --route_chan_width ${OPENFPGA_VPR_ROUTE_CHAN_WIDTH} --clock_modeling ideal ${OPENFPGA_VPR_EXTRA_OPTIONS}
+
+# Read OpenFPGA architecture definition
+read_openfpga_arch -f ${OPENFPGA_ARCH_FILE}
+
+# Read OpenFPGA simulation settings
+read_openfpga_simulation_setting -f ${OPENFPGA_SIM_SETTING_FILE}
+
+# Annotate the OpenFPGA architecture to VPR data base
+# to debug use --verbose options
+link_openfpga_arch --sort_gsb_chan_node_in_edges
+
+# Check and correct any naming conflicts in the BLIF netlist
+check_netlist_naming_conflict --fix --report ./netlist_renaming.xml
+
+# Optionally pb pin fixup
+${OPENFPGA_PB_PIN_FIXUP_COMMAND}
+
+# Apply fix-up to Look-Up Table truth tables based on packing results
+lut_truth_table_fixup
+
+# preload unique blocks from the provided xml file 
+read_unique_blocks --file ${READ_UNIQUE_BLOCKS_XML} --verbose --type xml
+
+# Build the module graph
+#  - Enabled compression on routing architecture modules
+#  - Enable pin duplication on grid modules
+build_fabric --compress_routing --group_tile ${OPENFPGA_GROUP_TILE_CONFIG_FILE} #--verbose
```

Comment:
Remove the ``--compress_routing`` option here. It may shadow the preloading.
Also if needed, create a new testcase where the ``--compress_routing`` is enabled.

Reply:
Double check: compress_routing is not only used for build_unique_modules, but also used in build_module_graph. Can it be dropped? 
![image](https://github.com/user-attachments/assets/cc8a73e5-457c-4edb-84cf-524f2a81cbd3)
![image](https://github.com/user-attachments/assets/0791244b-e002-4131-b808-70f5a008965a)

Reply:
will add a new test case to test it

Reply:
1. If compress_routing is not enabled and read_unique_blocks command is not called, the write_unique_blocks will not work because there is no unique blocks to write.
2. If compress_routing is not enabled and read_unique_blocks command is called, the write_unique_blocks will work. 
I will add a testcase of the second scenario to test its behaviour.

Reply:
Agree. The second scenario is what we should test. For the 1st scenario, the error message to be thrown may contain more information, e.g., ``To output unique blocks into a file, please run the command read_unique_block to preload the unique block data or enable compress_routing.